.github/workflows/driver-adapter-smoke-tests.yml

```diff
@@ -0,0 +1,77 @@
+name: Driver Adapters, Smoke Tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths-ignore:
+      - '.buildkite/**'
+      - '*.md'
+      - 'LICENSE'
+      - 'CODEOWNERS'
+      - 'renovate.json'
+
+jobs:
+  driver-adapter-smoke-tests:
+    name: Smoke tests for adapter ${{ matrix.adapter }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        adapter: ["neon:ws", "neon:http", planetscale, pg]
+  
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+
+    env:
+      JS_NEON_DATABASE_URL: ${{ secrets.JS_NEON_DATABASE_URL }}
+      JS_PLANETSCALE_DATABASE_URL: ${{ secrets.JS_PLANETSCALE_DATABASE_URL }}
+      JS_PG_DATABASE_URL: postgres://postgres:postgres@localhost:5432/test # ${{ secrets.JS_PG_DATABASE_URL }}
+
+    steps:
```

Comment:
I think `errors` test run is missing here - it uses fake adapter and therefore not a part of any other adapters test suite.